src/main/java/ua/danit/final_project/services/WorkCommentServiceImpl.java

```diff
@@ -3,6 +3,7 @@
 import org.joda.time.DateTime;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
 import ua.danit.final_project.configuration.StaticCollection;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'ua.danit.final_project.configuration.StaticCollection'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=88&issues=AWWVbxDV7khxiKmGg8xR&open=AWWVbxDV7khxiKmGg8xR)

[ISSUE_KEY]: AWWVbxDV7khxiKmGg8xR